src/disp_s1/product.py

```diff
@@ -559,54 +559,57 @@ def _create_grid_mapping(group, crs: pyproj.CRS, gt: list[float]) -> h5netcdf.Va
     return dset
 
 
-def create_compressed_products(comp_slc_dict: dict[str, Path], output_dir: Filename):
+def create_compressed_products(
+    comp_slc_dict: dict[str, list[Path]], output_dir: Filename
+):
     """Make the compressed SLC output product."""
 
     def form_name(filename: Path, burst: str):
         # filename: compressed_20180222_20180716.tif
-        date_str = _format_date_pair(*get_dates(filename.stem))
+        date_str = format_date_pair(*get_dates(filename.stem, fmt="%Y%m%d"))
```

Comment:
Do we want this to be `format_dates(*get_dates...)` now that there's 3?

Reply:
yes, thank you